bessctl/conf/perftest/flowgen.bess

```diff
@@ -32,7 +32,7 @@ flowgens = []
 for i in range(num_cores):
     bess.add_worker(wid=i, core=i)
     pkt_src = FlowGen(template = pkt_template, pps = init_pps / num_cores, flow_rate = num_flows / flow_time, flow_duration = flow_time, \
-        arrival = 'uniform', duration='uniform', quick_rampup = True)
+        arrival = 'uniform', duration='uniform', quick_rampup = True, burst = 32)
```

Comment:
Feel free to ignore this: we can skip this (in favor of brevity over completeness)